scripts/manage/push_config.sh

```diff
@@ -39,6 +39,17 @@ if [ $CURRENT_CONTEXT_CLUSTER != $GKE_CLUSTER ]; then
   exit 1
 fi
 
+source ~/spinnaker-for-gcp/scripts/manage/cluster_utils.sh
+
+CLUSTER_EXISTS=$(check_for_existing_cluster)
+
+if [ -z "$CLUSTER_EXISTS" ]; then
+  bold "Cluster $GKE_CLUSTER cannot be found. It may not exist."
+  bold "To recreate the cluster with this config, run:"
```

Comment:
Would running this script potentially do a lot more than recreating the cluster?

Reply:
The operations in setup.sh (enabling apis, installing to a cluster, setting up Redis, etc.) are designed to be idempotent for existing infrastructure. For infrastructure that has been lost, the script should succeed in restoring the requisites for a full installation. 

Next steps will involve testing and documenting the disaster and recovery modes for the full set of components. I'm also interested in introducing CI to automate our testing for these recovery tools.